be/src/www/doris.js

```diff
@@ -0,0 +1,148 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why not put all web to /webroot?

Reply:
I've moved all be's web files to /webroot/be